crates/wallet/src/db/balance.rs

```diff
@@ -0,0 +1,69 @@
+use crate::types::ProofState;
+use rusqlite::{Connection, Result};
+
+pub fn get_balance_for_node(conn: &Connection, node_id: u32) -> Result<Vec<(String, i64)>> {
```

Comment:
you are already in the file balance, so lets just call it `balance::get_for_node`